co.posit.rstudio.metainfo.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>co.posit.rstudio</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>AGPL-3.0-only</project_license>
+    <name>RStudio</name>
+    <summary>Integrated development environment for the R programming language</summary>
+    <description>
+        <p>
+            RStudio is an integrated development environment (IDE) for R, and
+            makes interacting with R quick and easy.
+        </p>
+        <p>
+            It includes a console, syntax-highlighting editor that supports
+            direct code execution, as well as tools for plotting, history,
+            debugging and workspace management.
+        </p>
+    </description>
+    <content_rating type="oars-1.1" />
+    <launchable type="desktop-id">co.posit.rstudio.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://iucar.fedorapeople.org/screenshots/rstudio/rstudio-ubuntu.png</image>
+        </screenshot>
+        <screenshot>
```

Comment:
there are too many screenshots and most seem to windows/mac. Is their any Linux screenshot?